keycloak/keycloak.config.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+set -eu -o pipefail
+
+KEYCLOAK_HOST=${KEYCLOAK_HOST:-"http://keycloak:8080/auth"}
+KEYCLOAK_CREATE_REALM=${KEYCLOAK_CREATE_REALM:-"admin"}
+KEYCLOAK_MASTER_USER=${KEYCLOAK_MASTER_USER:-"admin"}
+KEYCLOAK_MASTER_PASSWORD=${KEYCLOAK_MASTER_PASSWORD:-"admin"}
```

Comment:
hm... it sounds that will be loaded with admin/admin user. This being the case, could show a warning when the user loads the GUI/login screen.